Formula/zork.rb

```diff
@@ -0,0 +1,21 @@
+class Zork < Formula
+  desc "The Dugeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/a463c206cb0c714ba7483512a3a9edf543a7c2f5.tar.gz"
```

Comment:
Release cut at https://github.com/devshane/zork/releases/tag/v1.0.0, updated formula to use that instead.